packages/tpetra/core/src/Tpetra_MultiVector_def.hpp

```diff
@@ -971,8 +971,8 @@ namespace Tpetra {
     // Check whether the source object is a MultiVector.  If not, then
     // we can't even compare sizes, so it's definitely not OK to
     // Import or Export from it.
-    typedef MultiVector<Scalar, LocalOrdinal, GlobalOrdinal, Node> this_type;
-    const this_type* src = dynamic_cast<const this_type*> (&sourceObj);
+    typedef MultiVector<Scalar, LocalOrdinal, GlobalOrdinal, Node> MV;
+    const MV* src = dynamic_cast<const MV*> (&sourceObj);
```

Comment:
can we do this more simply with std::is_base_of or std::is_convertible?

Reply:
@tasmith4 I don't think so, since those both test for conditions on types at compile-time, and ``sourceObj`` could be a reference to anything in that inheritance tree at runtime.

These two changed lines weren't meant to be semantic changes and have nothing to do with the MV subview issue - I just wanted to get rid of warnings about ``this_type`` shadowing a member typedef.

Reply:
got it, thanks.  and sorry to have missed seeing your response.